test/unit/alphabet/composite/alphabet_variant_test.cpp

```diff
@@ -332,3 +333,31 @@ TEST(alphabet_variant_test, two_different_variants)
     seqan3::alphabet_variant<seqan3::rna4, seqan3::gap> r2{'A'_rna4};
     EXPECT_EQ(l, r2); // this works because rna4 and dna4 are implicitly convertible to each other
 }
+
+TEST(alphabet_variant_test, char_is_valid_for)
```

Comment:
Can you re-add the specific tests of the equivalent classes? I would always prefer if we have a generic implementation and one that validates that for specific values. Thank you :)

```c++
    EXPECT_TRUE(seqan3::char_is_valid_for<seqan3::gapped<seqan3::rna5>>('A')); // valid seqan3::rna5 char 
    EXPECT_TRUE(seqan3::char_is_valid_for<seqan3::gapped<seqan3::rna5>>('-')); // valid seqan3::gap char 
    EXPECT_FALSE(seqan3::char_is_valid_for<seqan3::gapped<seqan3::rna5>>('S')); // neither seqan3::rna5 nor seqan3::gap
```